# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsHTMLParts.h
# Commit: 2d16e9e90401
# Full Hash: 2d16e9e904010ba3389aad9729bac4f3744a588d
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 05:43:48
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 5b99d9a7ad8d -r 2d16e9e90401 layout/generic/nsHTMLParts.h
--- a/layout/generic/nsHTMLParts.h	Fri Jan 29 11:30:44 2021 +0000
+++ b/layout/generic/nsHTMLParts.h	Fri Jan 29 15:45:42 2021 +0000
@@ -138,8 +138,6 @@
                                  mozilla::ComputedStyle* aStyle);
 nsIFrame* NS_NewColorControlFrame(mozilla::PresShell* aPresShell,
                                   mozilla::ComputedStyle* aStyle);
-nsIFrame* NS_NewLegendFrame(mozilla::PresShell* aPresShell,
-                            mozilla::ComputedStyle* aStyle);
 nsIFrame* NS_NewTextControlFrame(mozilla::PresShell* aPresShell,
                                  mozilla::ComputedStyle* aStyle);
 nsContainerFrame* NS_NewListControlFrame(mozilla::PresShell* aPresShell,